### PR TITLE
improvement: allow multiple word submission + non latex questions + ignore deletion when message does not exist

### PR DESCRIPTION
**Pull request by Tran-Antoine**

### CODE REVIEW

File: discord-impl/src/main/java/net/starype/quiz/discordimpl/game/DiscordGameServer.java [hunk 1]
```diff
@@ -88,7 +91,12 @@ public void onPlayerScoreUpdated(Player<?> player) {
 
     @Override
     public void onQuestionReleased(Question question) {
-        sendAsLatexFile(question.getRawQuestion());
+        String rawQuestion = question.getRawQuestion();
+        if(question.isTagAttached("Math")) {
```
Comment (mredstone6) [inline discord-impl/src/main/java/net/starype/quiz/discordimpl/game/DiscordGameServer.java:95]: I don't believe that the formatting of the answer as `LaTeX` or `Default` should be decide on whether or not the question possesses a given tag ! We could either add a new properties in the `TOML` or detect certain characteristic in the text (such as `"\text",```java ` ). We could also add a keyword in the answer that decide the type of formatting (such as if answer starts with `"LaTeX"` format as LaTeX) or even better parse only certain area as so.

File: discord-impl/src/main/java/net/starype/quiz/discordimpl/command/RoundAddCommand.java [hunk 2]
```diff
@@ -69,23 +70,35 @@ public void execute(CommandContext context) {
...
+                () -> args.length >= 3 && asRoundCount(args[2]).isEmpty(),
+                "Second argument must be a number between 1 and 20");
 
         return conditions;
     }
 
+    private static Optional<Integer> asRoundCount(String arg) {
+        try {
+            int value = Integer.parseInt(arg);
+            if(value > 0 && value < 21) {
```
Comment (mredstone6) [inline discord-impl/src/main/java/net/starype/quiz/discordimpl/command/RoundAddCommand.java:85]: We could introduce dedicated constant for the maximum amount of round that could be created

**Review by mredstone6 — CHANGES_REQUESTED** [review]

File: discord-impl/src/main/java/net/starype/quiz/discordimpl/game/DiscordGameServer.java (hunk 1 above)
Comment (mredstone6) [reply discord-impl/src/main/java/net/starype/quiz/discordimpl/game/DiscordGameServer.java:95]: Thought it is a lot of work and could be achieve in another PR. Yet, I don't think we should merge before fixing that. But it is possible to merge first if you just want the code as fast as possible on the `master`

File: discord-impl/src/main/java/net/starype/quiz/discordimpl/command/RoundAddCommand.java (hunk 2 above)
Comment (Tran-Antoine) [reply discord-impl/src/main/java/net/starype/quiz/discordimpl/command/RoundAddCommand.java:85]: Yes indeed

File: discord-impl/src/main/java/net/starype/quiz/discordimpl/game/DiscordGameServer.java (hunk 1 above)
Comment (Tran-Antoine) [reply discord-impl/src/main/java/net/starype/quiz/discordimpl/game/DiscordGameServer.java:95]: I agree, let's make another PR for this. We will wait before merging this one

**Review by mredstone6 — APPROVED** [review]

**Pull request merged**